conf/repos.yaml.template

```diff
@@ -35,3 +35,290 @@ REPOS:
   # which are essentially required for generating swid tags in RHEL8 content host.
   SWID_TOOLS_REPO: replace-with-swid-tools-repo
   FAKE_REPO_ZOO3: replace-with-zoo-fedorapeople-repo
+
+# Robottelo fixtures repos
+  MODULE_STREAM_0:
+    URL:
+    ERRATA:
+      - "EX#"
```

Comment:
Any reason we are adding the duplicate `EX#`, As this is template file we can avoid that ?

Reply:
For some tests we only pull out one errata id like `settings.repos.yum_0.errata[2]`. The checks for GitHub were failing when these were None.

Reply:
ohh.. okay